### PR TITLE
Reverted templates to original versions.

### PR DESCRIPTION
**Pull request by apecoraro**
I mistakenly used the templates from the latest version of Swagger
Codegen for my last commit, which ended up breaking things. So these are
the same changes on the templates to fix the /nfs/aliases/<AID> end
point, but applied to the version of the templates that we use in our
build (1939ce8e91f9de4a35ba3d90fd99a28f41c5035c).

### CODE REVIEW

**Comment by apecoraro** [comment]
This looks like a lot of changes, but if there were some way to compare this to versions of the templates prior to my last commit you'd see that the only changes the ones related to the vendor extension code.

File: swagger_templates/python/api.mustache [hunk 1]
```diff
@@ -126,66 +107,52 @@ class {{classname}}(object):
 {{/vendorExtensions.x-isi-url-encode-path-param}}
```
Comment (apecoraro) [inline swagger_templates/python/api.mustache]: This is the portion of code that I added for tagging a parameter with "__x-isi-url-encode-path-param__", the rest of these changes are due to reverting from the latest upstream version of the template to the template that we were using prior to my last commit.

File: swagger_templates/python/api_client.mustache [hunk 2]
```diff
@@ -74,6 +82,11 @@ class ApiClient(object):
         self.cookie = cookie
         # Set default User-Agent.
         self.user_agent = '{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{packageVersion}}}/python{{/httpUserAgent}}'
+        # This is used for detecting for the special case of a path parameter
```
Comment (apecoraro) [inline swagger_templates/python/api_client.mustache]: This code was added to help with detecting the "__x-isi-url-encode-path-param__". This is slightly different than before. It used to do this inside the path parameter loop, but this is more efficient.

File: swagger_templates/python/api_client.mustache [hunk 3]
```diff
@@ -95,55 +108,46 @@ class ApiClient(object):
...
+                v_str = str(self.to_path_value(v))
                 # Check for the special case of the
                 # x-isi-url-encode-path-param, which indicates that the
                 # parameter should be encoded with quote_plus in order
                 # to encode the '/' character.
-                quote_plus_tag = "__x-isi-url-encode-path-param__"
-                tag_len = len(quote_plus_tag)
                 # check if the first part of v_str matches the tag
-                if v_str[:tag_len] == quote_plus_tag:
+                if v_str[:self.quote_plus_tag_len] == self.quote_plus_tag:
```
Comment (apecoraro) [inline swagger_templates/python/api_client.mustache]: This is where it loops through the path parameters looking for the "__x-isi-url-encode-path-param__" tag and calling quote_plus on the value if it finds it.

File: swagger_templates/python/api_client.mustache [hunk 4]
```diff
@@ -95,55 +108,46 @@ class ApiClient(object):
...
+                    replacement = quote(v_str)
+                resource_path = resource_path.\
+                    replace('{' + k + '}', replacement)
 
         # query parameters
         if query_params:
             query_params = self.sanitize_for_serialization(query_params)
-            query_params = self.parameters_to_tuples(query_params,
-                                                     collection_formats)
+            query_params = {k: self.to_path_value(v)
```
Comment (apecoraro) [inline swagger_templates/python/api_client.mustache]: These other changes are just due to the fact that I used the wrong template in my last commit.

File: swagger_templates/python/model.mustache [hunk 5]
```diff
@@ -75,35 +75,6 @@ class {{classname}}(object):
         :type: {{datatype}}
         """
         {{#isEnum}}allowed_values = [{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]
-{{#isContainer}}
```
Comment (apecoraro) [inline swagger_templates/python/model.mustache]: Again, these changes were introduced when I upgraded our original template to the latest version of the template from the upstream Swagger. So now they are being removed by moving back to the version of the template that comes with the version of Swagger Codegen that we use for our build.

**Comment by cbrainerd** [comment]
LGTM 👍

**Pull request merged**